cli50

```diff
@@ -31,7 +32,40 @@ def main():
     if not args["fast"]:
         subprocess.call(["docker", "pull", "cs50/cli"])
 
-    # mount directory in CS50 CLI
+    # check for running containers
+    stdout = subprocess.check_output([
+        "docker", "ps",
+            "--all",
+            "--filter",
+            "volume={}".format(directory),
+            "--format", "{{.ID}}\t{{.Image}}\t{{.RunningFor}}"
+            ]).decode("utf-8")
+    containers = []
+    for line in stdout.rstrip().splitlines():
+        ID, Image, RunningFor = line.split("\t")
+        if Image == "cs50/cli":
```

Comment:
@dmalan, any reason not to drop `{{.Image}}`, and `if Image == "cs50/cli"`, and add `--filter ancestor=cs50/cli`?

https://docs.docker.com/engine/reference/commandline/ps/#formatting